user/caribou/module/src/AD9249EventConverter.cc

```diff
@@ -224,12 +265,12 @@ bool AD9249Event2StdEventConverter::Converting(
   d2->AddPlane(plane);
 
   // use timestamps
-  if( m_useTime ){
+  if (m_useTime) {
```

Comment:
Is this still necessary of superseded by the new upstream functionality https://gitlab.cern.ch/corryvreckan/corryvreckan/-/blob/master/src/modules/EventLoaderEUDAQ2/README.md?plain=1#L99 ?

Reply:
Yes it is. It will change the default behavior, but since this affects only us I guess it is fine.